modules/gdscript/gdscript_vm.cpp

```diff
@@ -688,7 +689,17 @@ Variant GDScriptFunction::call(GDScriptInstance *p_instance, const Variant **p_a
 				GET_INSTRUCTION_ARG(b, 1);
 				GET_INSTRUCTION_ARG(dst, 2);
 
-				operator_func(a, b, dst);
+				operator_func(a, b, dst, valid);
+
+				if (!valid) {
+					if ((*dst).get_type() == Variant::STRING) {
+						err_text = (*dst);
+						err_text += " in operator '" + Variant::get_operator_name((Variant::Operator)operator_idx) + "'.";
+					} else {
+						err_text = "Invalid operands '" + Variant::get_type_name(a->get_type()) + "' and '" + Variant::get_type_name(b->get_type()) + "' in operator '" + Variant::get_operator_name((Variant::Operator)operator_idx) + "'.";
+					}
+					OPCODE_BREAK;
+				}
```

Comment:
Should this also be inside `#ifdef DEBUG_ENABLED` like `OPCODE_OPERATOR` (just above this one)?

Reply:
I guess...